modules/rdcb/main.tf

```diff
@@ -51,7 +51,7 @@ resource "aws_security_group" "rdcb-sg1" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
-  tags {
+  tags = {
```

Comment:
```suggestion
  tags = {
    merge(map("Name", "${var.StackName}-rdcb-sg1"), var.tags)
  }
```

Reply:
Going to break this out into a separate task, and address all of the resources